pages/how-to-contribute.md

```diff
@@ -19,9 +19,19 @@ This document provides basic instructions for contributing to Accumulo.  If you
 
 ## Issues
 
-Any contribution should have a corresponding issue. Accumulo uses [JIRA] for issue tracking. Before creating an issue,
-you will need to create an [Apache JIRA account][jira-signup]. If you need help finding an issue to work on, check out
-the [open issues labeled for newbies][newbie-issues] or [contact us][contact].
+Accumulo uses GitHub issues to track bugs and features.  Each git repository
+has its own issues.  In GitHub pull request are issues, therefore creating an
+issue before a pull request is optional. If unsure wether to start with an
+issue or pull request, then create an issue.  For each repository,
+CONTRIBUTING.md may contain information about issues specific to it. If you
+need help finding an issue to work on, check out the
+[open issues labeled 'helpwanted'][helpwanted] or [contact us][contact].
+
+Accumulo previously used [JIRA] for issue tracking, but is now transitioning to
+GitHub issues.  All new issues should be opened in GitHub.  Existing issues in
+JIRA can still be updated.  Eventually JIRA will be transitioned to a read-only
+state for reference.  For finding issues to work, there may still be [open
+issues labeled for newbies][newbie-issues] in JIRA.
 
 ## Repositories
```

Comment:
Could add links to all issues trackers in the table in this section

Reply:
There is a small typo in the sentence "If unsure wether to start with an issue or pull request, then create an issue". 'wether' should be 'whether'.